includes/admin/feedzy-wp-widget.php

```diff
@@ -37,7 +37,7 @@ class feedzy_wp_widget extends WP_Widget {
 	 * @param   Feedzy_Rss_Feeds_Admin $plugin_admin The Feedzy_Rss_Feeds_Admin object.
 	 */
 	public function __construct( $plugin_admin = null ) {
-		parent::__construct( false, $name = __( 'Feedzy RSS Feeds', 'feedzy-rss-feeds' ) );
+		parent::__construct( false, $name = FEEDZY_NAME );
```

Comment:
Is this change related to the problem regarding the text-domain loading in WP 6.7?

Reply:
@Soare-Robert-Daniel Yes

Reply:
I've utilized this PHP [constant](https://github.com/Codeinwp/feedzy-rss-feeds/blob/master/feedzy-rss-feed.php#L126) here to avoid translating the string again.

Reply:
One problem here is that the name is a user-facing string, and it will need to be translatable since we have an initiative with this (like https://github.com/Codeinwp/feedzy-rss-feeds/pull/997/files)

Can this initialization https://github.com/Codeinwp/feedzy-rss-feeds/blob/a438e1661f3a1a2d33ccd254ae77363513e9d4a6/includes/feedzy-rss-feeds.php#L232-L233 be moved in a context that allows translations?

Reply:
@Soare-Robert-Daniel I’ve updated the widgets_init method. Could you please review my latest commit?